src/components/Promo/PromoCard.tsx

```diff
@@ -0,0 +1,101 @@
+import { Button } from '@chakra-ui/react'
+import dayjs from 'dayjs'
+import isBetween from 'dayjs/plugin/isBetween'
+import { useCallback, useMemo } from 'react'
+import { useHistory } from 'react-router-dom'
+import FoxyPromo from 'assets/foxy-promo.jpg'
+import { Card } from 'components/Card/Card'
+import { Carousel } from 'components/Carousel/Carousel'
+import { RawText } from 'components/Text'
+import { WalletActions } from 'context/WalletProvider/actions'
+import { useWallet } from 'hooks/useWallet/useWallet'
+dayjs.extend(isBetween)
+
+const promoData = [
+  {
+    title: 'Earn 3.15% APY on FOX',
+    body: 'Your FOX is put to work across different DeFi protocols to earn the best yield possible.',
+    cta: 'Deposit FOX Now',
+    image: `url(${FoxyPromo})`,
+    colorScheme: 'pink',
+    startDate: '2022-11-28 8:00 AM',
+    endDate: '2022-12-4 8:00 AM',
+    id: 'foxy-promo',
+    walletRequired: true,
+    href: '?chainId=eip155%3A1&contractAddress=0xee77aa3Fd23BbeBaf94386dD44b548e9a785ea4b&assetReference=0xc770eefad204b5180df6a14ee197d99d808ee52d&rewardId=0xDc49108ce5C57bc3408c3A5E95F3d864eC386Ed3&provider=ShapeShift&modal=deposit',
+  },
+]
+
+export const PromoCard = () => {
+  const {
+    dispatch,
+    state: { isDemoWallet, isConnected },
+  } = useWallet()
+  const history = useHistory()
+
+  const handleWalletModalOpen = useCallback(
+    () => dispatch({ type: WalletActions.SET_WALLET_MODAL, payload: true }),
+    [dispatch],
+  )
+
+  const handleClick = useCallback(
+    ({ href, walletRequired }: { href: string; walletRequired: boolean }) => {
+      if (walletRequired) {
+        if (isConnected && !isDemoWallet) {
```

Comment:
We might want to add chain support detection
<img width="1086" alt="image" src="https://user-images.githubusercontent.com/17035424/204401598-abb50871-23f8-4a98-9600-cee44042563c.png">

See:

https://github.com/shapeshift/web/blob/f03e7a8c7a9a0b538bf4d47880279a911f35522b/src/plugins/foxPage/foxPage.tsx#L151-L153